forward/events.go

```diff
@@ -0,0 +1,64 @@
+package forward
+
+// An EventListener handles events given to it by the SWIM node. HandleEvent should be thread safe.
+type EventListener interface {
+	HandleEvent(interface{})
```

Comment:
I agree that we should share the same interface in the entire project. I am not a fan of the state of how we handle events in ringpop-go right now so I think I will do some work on them. For now, I think it's best to use a shared interface.